ecs/basicsystem.go

```diff
@@ -0,0 +1,79 @@
+package ecs
+
+import "runtime"
+
+// BasicSystemWrapper is the default implementation of the System interface.
+type BasicSystemWrapper struct {
```

Comment:
maybe this wrapper is not necessary. In Java such classes would be named Adapter but some people call them Stub as well.

SystemStub or SystemAdapter.